pandas/tests/extension/decimal/array.py

```diff
@@ -155,6 +155,14 @@ def __setitem__(self, key, value):
     def __len__(self) -> int:
         return len(self._data)
 
+    def __contains__(self, item) -> bool:
+        if not isinstance(item, decimal.Decimal):
```

Comment:
does this have actual tests?

Reply:
Yes, in `pandas/tests/extension/decimal/test_decimal.py` there is `TestInterface`, which inherits from `BaseInterfaceTests`.